### PR TITLE
adding theme suggestions input alter

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/pineapple.theme [hunk 1]
```diff
@@ -33,6 +33,20 @@ function pineapple_theme_suggestions_field_alter(array &$suggestions, array $var
...
+ * Implements hook_theme_suggestions_input_alter().
+ */
+function pineapple_theme_suggestions_input_alter(array &$suggestions, array $variables) {
+  $element = $variables['element'];
+  if (!empty($element['#name'])) {
+    $suggestions[] = 'input-' .$element['#name'];
+  }
+  if (!empty($element['#webform_id'])) {
+    $suggestions[] = 'input-' . $element['#webform_id'] . '-' . $element['#name'];
+    $suggestions[] = 'input-' . $element['#webform_id'] . '-' . $element['#type'];
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/pineapple.theme:46]: Use underscores not hyphens.

Comment (ElXyfer) [inline web/themes/custom/pineapple/pineapple.theme:46]: Type should be a different if statement and you should be checking if both variables used inside the if statement are not empty

Comment (katlazarova) [reply web/themes/custom/pineapple/pineapple.theme]: Done

Comment (katlazarova) [reply web/themes/custom/pineapple/pineapple.theme]: Done

File: web/themes/custom/pineapple/pineapple.theme [hunk 2]
```diff
@@ -33,6 +33,22 @@ function pineapple_theme_suggestions_field_alter(array &$suggestions, array $var
...
+function pineapple_theme_suggestions_input_alter(array &$suggestions, array $variables) {
+  $element = $variables['element'];
+  if (!empty($element['#name'])) {
+    $suggestions[] = 'input_' .$element['#name'];
+  }
+  if (!empty($element['#webform_id']) && !empty($element['#name'])) {
+    $suggestions[] = 'input_' . $element['#webform_id'] . '_' . $element['#name'];
+  }
+  if (!empty($element['#webform_id']) && !empty($element['#type'])) {
+    $suggestions[] = 'input_' . $element['#webform_id'] . '_' . $element['#type'];
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/pineapple.theme:48]: Add double underscores and put webform as opposed to the middle

File: web/themes/custom/pineapple/pineapple.theme [hunk 3]
```diff
@@ -33,6 +33,22 @@ function pineapple_theme_suggestions_field_alter(array &$suggestions, array $var
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_input_alter().
+ */
+function pineapple_theme_suggestions_input_alter(array &$suggestions, array $variables) {
+  $element = $variables['element'];
+  if (!empty($element['#name'])) {
+    $suggestions[] = 'input_' .$element['#name'];
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/pineapple.theme:42]: add a space after the full stop

File: web/themes/custom/pineapple/pineapple.theme (hunk 2 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/pineapple.theme:48]: Sorry, I don't understand.

File: web/themes/custom/pineapple/pineapple.theme (hunk 3 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/pineapple.theme]: Done.

File: web/themes/custom/pineapple/pineapple.theme (hunk 2 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/pineapple.theme]: Done

**Pull request merged**